### PR TITLE
add hugo documentation for starting golden seed locally

### PR DESCRIPTION
**Pull request by jacobkagon**
### Master Pivotal ticket
* https://www.pivotaltracker.com/n/projects/2481179/stories/178952981
### Child Redmine ticket(s)
* ()
* ()

### Local build result

```
bundle exec rake parallel:spec[4]
bundle exec cucumber
```

### Latest rebase/merge tag
*

---

### Data ticket(s) Followup
* (redmine links here - optional)

### Related Pull Requests
* (github links here - optional)

---

#### Peer Review
* (For code review)

#### Functional Testing
* (For testing locally)

#### Deployment
* (for release manager and/or build manager)

### CODE REVIEW

File: hugo/content/docs/data_seed.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+3.  In a third terminal window, start sidekiq: `bundle exec sidekiq start`
+
+4.  \*Make sure there is enough memory on the server to run both of those commands
+
+5.  Open Enroll in your browser by going to: **http://localhost:3000/** if running locally
+
+6.  Log in as hbx admin
+
+7.  Go to admin dropdown menu and click goldenseed
```
Comment (ZASMan) [inline hugo/content/docs/data_seed.md:23]: I think it actually says "Data Seed UI" or something. Double check.

File: hugo/content/docs/data_seed.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+1.  Start server: `rails s`
+
+2.  Start redis-server in another terminal. To do this run: `redis-server`
+
+        If redis is not found, run: ```brew install redis```
+
+3.  In a third terminal window, start sidekiq: `bundle exec sidekiq start`
+
+4.  \*Make sure there is enough memory on the server to run both of those commands
```
Comment (ZASMan) [inline hugo/content/docs/data_seed.md:17]: there's an extra \ on this line, and you put a *before "make", a single astereik is for italicizing. Did you mean to put the single asteriks around "Make sure"? Maybe do that and get rid of the slash.

**Review by ZASMan — CHANGES_REQUESTED** [review]

**Review by ZASMan — APPROVED** [review]
Thanks for taking care of those!

**Review by bill-transue — APPROVED** [review]
This is great however we also want to document the `rake` task methods, which are faster and more useful for developers. I'll have a separate ticket made for that. Thanks @jacobkagon!

**Pull request merged**